src/main/java/com/googleinterns/zoomtube/servlets/DeleteTranscriptServlet.java

```diff
@@ -31,7 +31,7 @@
 @WebServlet("/delete-transcript")
 public class DeleteTranscriptServlet extends HttpServlet {
   private static String HOME_PATH = "/";
-  
+
```

Comment:
this should have been done in its own PR as it is unrelated.

Reply:
The linter did this

Reply:
It is weird it didn't fix this before it was merged.